package.json

```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "asar": "^0.11.0",
     "electron-download": "^2.0.0",
-    "electron-osx-sign": "^0.3.0",
+    "electron-osx-sign": "^0.4.0-beta3",
```

Comment:
0.4.0-beta3 introduces entitlements check so issue similar to #385 won't appear again in `electron-osx-sign`.